### ISSUE TITLE
Support library translations (name, descriptions, images of shows/movies...)

### ISSUE THREAD

**Issue by zoriya** [zoriya/Kyoo#87]
Titles and descriptions should be translatable and translations should be stored in the database.

**Comment by bo0tzz** [comment]
https://weblate.org/ is a great tool for creating translations, and they offer a free cloud plan for open-source projects.

**Comment by zoriya** [comment]
No, this issue is for fetching metadata in multiples languages from themoviedb and allow users to select which version they want. This includes things like titles, poster or descriptions of shows/movies...
The web/android app already supports translations, it uses simple json (stored in https://github.com/zoriya/Kyoo/tree/master/front/translations) for now tho. I could be nice to use a service for that

**Comment by bo0tzz** [comment]
My bad, I misunderstood the feature description!

**Comment by zoriya** [comment]
It was a pretty bad name, that's fair

**Comment by zoriya** [comment]
Implemented in master, will be released with v5!
